src/Agent.Worker/JobRunner.cs

```diff
@@ -10,100 +10,149 @@ namespace Microsoft.VisualStudio.Services.Agent.Worker
     [ServiceLocator(Default = typeof(JobRunner))]
     public interface IJobRunner : IAgentService
     {
-        Task<int> RunAsync(JobRequestMessage message);
+        Task RunAsync(JobRequestMessage message);
     }
 
     public sealed class JobRunner : AgentService, IJobRunner
     {
-        public async Task<int> RunAsync(JobRequestMessage message)
+        public async Task RunAsync(JobRequestMessage message)
```

Comment:
I removed the return type. Exit codes are a concept for the host later - not the job runner class. I talked with Ting, he will add a return type back to this method (TaskResult of the job) and it will be the host layer's problem to communicate that back across process boundary. Since TaskResult is an int (enum) it probably makes since to just cast it and use that as the exit code... Ting will figure it out. Point here is that job runner shouldn't know anything about exit codes.